NEWS.md

```diff
@@ -0,0 +1,14 @@
+ecoretriever 0.2
+================
+
+NEW FEATURES
+* We added a new function `get_updates` which can be used to update the `retriever` scripts. This is a big improvement for users becuase the past behavior everytime the package was loaded it updated its scripts which resulted in annoying lag times. The log of the scripts update can be printed pretty easily as well. 
```

Comment:
"becuase" -> "because"

"the past behavior everytime the package was loaded it updated its scripts which resulted in annoying lag times." 
->
"it avoids automatically updating the scripts every time the package is imported."